server/server/app.py

```diff
@@ -34,7 +35,9 @@ def app_factory() -> Tuple[Api, Flask]:
     api.add_resource(Images, '/images/<string:division>/<int:vol>')
     api.add_resource(Epigraphs, '/epigraphs')
     api.add_resource(WhyWeRead, '/whyweread')
-    api.add_resource(Glossary, '/glossary/<string:glossword>')
+    api.add_resource(Glossary, '/glossary')
+    api.add_resource(DictionaryAdjacent, '/dictionary_adjacent/<string:word>')
+    api.add_resource(DictionarySimilar, '/dictionary_similar/<string:word>')
```

Comment:
Hmm, `adjacent` and `similar` are subtypes of `dictionary`, so properly addressed they should look like this:
```
api.add_resource(DictionaryAdjacent, '/dictionary/adjacent/<string:word>')
api.add_resource(DictionarySimilar, '/dictionary/similar/<string:word>')
```

Reply:
They are subtypes of `dictionary_full`, not of `dictionary` but yet, point taken and will change that.

Reply:
[REST resource naming](http://www.restapitutorial.com/lessons/restfulresourcenaming.html) is a large and rather difficult subject, but it really pays off to have a proper RESTful API.